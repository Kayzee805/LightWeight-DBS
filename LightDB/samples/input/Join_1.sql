SELECT * FROM Students, Enrollments WHERE Students.id = Enrollments.student_id
