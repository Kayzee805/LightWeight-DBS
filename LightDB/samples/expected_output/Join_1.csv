100,7,4,100,301
101,8,3,101,202
101,8,3,101,301
102,8,2,102,202
102,8,2,102,302
103,10,3,103,305
103,10,3,103,303
104,11,4,104,304
104,11,4,104,305

